test/pgtest/transactions.pt

```diff
@@ -477,16 +477,7 @@ ReadyForQuery {"status":"I"}
 # Verify implicit transactions are properly upgraded
 send
 Query {"query": "CREATE TABLE t (a INT)"}
-Parse {"query": "INSERT INTO t VALUES (1)"}
-Bind
-Execute
-Parse {"query": "BEGIN"}
-Bind
-Execute
-Parse {"query": "COMMIT"}
-Bind
-Execute
-Sync
+Query {"query": "INSERT INTO t VALUES (1); BEGIN; COMMIT;"}
```

Comment:
The comment for this test is now wrong, since no txn upgrade occurs.

Reply:
I'm not exactly sure what this is testing, but the same upgrade logic as before should still be happening, just using the simple protocol. That is:
1. `INSERT` starts an implicit transaction.
2. `BEGIN` upgrades the implicit transaction to an explicit transaction.
3. `COMMIT` commits the explicit transaction.

Unless I don't fully understand the simple protocol.

Reply:
Ah I didn't notice that all the statements got merged into one. And it looks like the lines being removed here got added to a new file in pgtest-mz, so this is good.